components/lib/crash/src/main/res/values/strings.xml

```diff
@@ -24,6 +24,9 @@
     <!-- Label of notification showing that the crash report service is running. %1$s will be replaced with the name of the organization (e.g. Mozilla). -->
     <string name="mozac_lib_send_crash_report_in_progress">Sending crash report to %1$s</string>
 
+    <!-- Label of notification showing that the telemetry report service is running. %1$s will be replaced with the name of the organization (e.g. Mozilla). -->
+    <string name="mozac_lib_send_telemetry_report_in_progress">Sending telemetry report to %1$s</string>
```

Comment:
I think "sending" could still be misleading. If telemetry is disabled then I think we'd still start this service, but Glean itself would not send any data. Maybe let's say something about us collecting the data?

Reply:
That's true.  We are just collecting the data, not technically sending it.  I'll update.